docker/versions.dockerfile

```diff
@@ -17,7 +17,7 @@ ARG NPM_VERSION=10.7.0
 ARG NUGET_INSPECTOR_VERSION=0.9.12
 ARG PHP_VERSION=8.1
 ARG PIPTOOL_VERSION=24.0
-ARG PNPM_VERSION=8.10.3
+ARG PNPM_VERSION=9.2.0
```

Comment:
Commit message nit: "for unknown reason" -> "for unknown reason*s*" or "for *an* unknown reason"